Model/Service/Facet/BuildWebFacetService.php

```diff
@@ -290,6 +290,28 @@ private function makeArrayFromValue($name, $value): array
             return $value;
         }
 
-        throw new NostoException(sprintf('Can not get value for filter: %s', $name));
+        // @codingStandardsIgnoreStart
+        $this->logger->debugWithSource(
+            sprintf(
+                'Can not get value for filter: %s. Value passed was %v (type of %t, class - %c)',
+                $name,
+                $value,
+                gettype($value),
+                (gettype($value) == 'object') ? get_class($value) : 'not an object'
+            ),
+            [],
+            $this
+        );
+
+        throw new NostoException(
+            sprintf(
+                'Can not get value for filter: %s. Value passed was %v (type of %t, class - %c)',
+                $name,
+                $value,
+                gettype($value),
+                (gettype($value) == 'object') ? get_class($value) : 'not an object'
+            )
+        );
+        // @codingStandardsIgnoreEnd
```

Comment:
@olsi-qose adding ignore statements did the trick but not sure how acceptable that is?